src/ui/widgets/GroupBox/__snapshots__/groupBox.test.tsx.snap

```diff
@@ -1,13 +1,20 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<GroupingContainerComponent /> it matches the snapshot 1`] = `
-<fieldset>
+<fieldset
+  style={
+    Object {
+      "backgroundColor": undefined,
```

Comment:
I guess the background colour prop isn't defined in the snapshot test?
Probably a good idea to do so

Reply:
Good idea, done.